test/testautomation_audio.c

```diff
@@ -30,8 +31,38 @@ static void audioSetUp(void *arg)
     }
 }
 
+static void audioSetUp(void *arg)
+{
+    g_audioInitCount = 0;
+    g_originalAudioDriver = SDL_GetCurrentAudioDriver();
+
+    /* Stop SDL audio subsystem */
+    while (SDL_WasInit(SDL_INIT_AUDIO)) {
+        SDL_QuitSubSystem(SDL_INIT_AUDIO);
+        SDLTest_AssertPass("Call to SDL_QuitSubSystem(SDL_INIT_AUDIO)");
+        g_audioInitCount += 1;
+    }
```

Comment:
`SDL_WasInit` is used here to initialize the refcount;

The unit tests already do `SDL_QuitSubSystem(SDL_INIT_AUDIO)` + `SDL_SetHint(SDL_HINT_AUDIO_DRIVER, xxx)` + `SDL_InitSubSystem(SDL_INIT_AUDIO)`, but I noticed the hint was not applied.